server/test/api/depots.js

```diff
@@ -0,0 +1,108 @@
+/* jshint expr: true */
+/* global describe, it, beforeEach */
```

Comment:
You do not need to this comment starting with `global`.  JsHint is now aware of mocha's global variables.